src/middleware/graphqlServer.js

```diff
@@ -0,0 +1,67 @@
+const {ApolloServer} = require('apollo-server-express');
+// import loginWithToken from "../users/token";
+// import {configuration as corsConfiguration} from "../../middleware/cors";
+
+const {
+    ApolloServerPluginLandingPageGraphQLPlayground,
+    // ApolloServerPluginLandingPageDisabled
+} = require('apollo-server-core');
+
+const graphql = async () => {
+    const typeDefs = `
+            type Query {
+                totalPosts: Int!
+            }
+        `;
+
+    //resolvers
+    const resolvers = {
+        Query: {
+            totalPosts: () => 42,
+        },
+    };
+    const server = new ApolloServer(
+        {
+            typeDefs,
+            resolvers,
+            plugins: [
+                // eslint-disable-next-line new-cap
+                ApolloServerPluginLandingPageGraphQLPlayground(),
+                // ApolloServerPluginLandingPageDisabled()
+            ],
+            context: async ({req, res}) => {
+                // const token = req?.cookies["app_login_token"];
+
+                const context = {
+                    req,
+                    res,
+                    user: {},
+                };
+
+                // const user = token ? await loginWithToken({token}) : null;
```

Comment:
Could we remove the commented out code if its not needed?

Reply:
we will add this back in when we implement auth soon.